client/stylesheets/_modal.scss

```diff
@@ -1034,6 +1034,34 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Join Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Join Button ~~~~*/
+
+.modJoinTrigger {
+     color: #fff !important;
+     border: 1px solid #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 5 spaces